### ISSUE TITLE
Colocalisation example image data

### ISSUE THREAD

**Issue by tischi** [NEUBIAS/training-resources#633]
Hi @tibuch @stemarcotti @ssgpers @manerotoni @AnniekStok ,

Would some of you have image data that has:

- two channels
- there is partial colocalisation of the signal in the two channels
- there are only few saturated pixels
- ideally, the images are spatially calibrated

**Comment by AnniekStok** [comment]
Will this do? It is spatially scaled, but very anisotropic. 

[xyzc_8bit_nuclei.tif.gz](https://github.com/NEUBIAS/training-resources/files/14773665/xyzc_8bit_nuclei.tif.gz)

**Comment by AnniekStok** [comment]
Yes sorry, I noticed it later and tried to find a better region, but couldn't find one. I looked further and found a different image that does not have this problem, is this useful? 
[xyzc_8bit_nuclei2.tif.gz](https://github.com/NEUBIAS/training-resources/files/14776072/xyzc_8bit_nuclei2.tif.gz)

**Comment by manerotoni** [comment]
Are you thinking of small or large colocalization objects?
I can find images of reporters that are in the nucleus and/or cytoplasm. This is a spatially quite big.
We had a project with vesicle STED colocalization, although published it is not my data.

**Comment by tischi** [comment]
> We had a project with vesicle STED colocalization, although published it is not my data.

@manerotoni  That would be cool! If it is published the data should be available? Maybe I can find it myself if you point me to the publication?

> I looked further and found a different image that does not have this problem, is this useful?

This is better in terms of the saturation issue, but something with vesicular structures that partially co-localise would be more ideal. The "real data" [here](https://github.com/embl-cba/imagej-courses/tree/master/data/colocalization) would be great, but it does have the problem of saturated signal in one of the channels.

**Comment by ssgpers** [comment]
Here are some data on ER exit sites:
https://oc.embl.de/index.php/s/54DLd5a7GODpIsg
Maybe this helps
